js/whitebit.js

```diff
@@ -615,11 +710,439 @@ module.exports = class whitebit extends Exchange {
+            if (uniqueId === undefined) {
+                throw new ArgumentsRequired (this.id + ' fetchDepositAddress() requires an uniqueId when the ticker is fiat');
+            }
+        }
+        const response = await this[method] (this.extend (request, params));
+        // Response for fiat
+        // {
+        //     "url": "https://someaddress.com"                  // address for deposit
+        // }
+        // Response for crypo
+        // {
+        //     "account": {
+        //         "address": "GDTSOI56XNVAKJNJBLJGRNZIVOCIZJRBIDKTWSCYEYNFAZEMBLN75RMN",        // deposit address
+        //         "memo": "48565488244493"                                                      // memo if currency requires memo
+        //     },
+        //     "required": {
+        //         "fixedFee": "0",                                                              // fixed deposit fee
+        //         "flexFee": {                                                                  // flexible fee - is fee that use percent rate
+        //             "maxFee": "0",                                                            // maximum fixed fee that you will pay
+        //             "minFee": "0",                                                            // minimum fixed fee that you will pay
+        //             "percent": "0"                                                            // percent of deposit that you will pay
+        //         },
+        //         "maxAmount": "0",                                                             // max amount of deposit that can be accepted by exchange - if you deposit more than that number, it won't be accepted by exchange
+        //         "minAmount": "1"                                                              // min amount of deposit that can be accepted by exchange - if you will deposit less than that number, it won't be accepted by exchange
+        //     }
+        // }
+        let memo = undefined;
+        let address = this.safeValue (response, 'url');
+        if (address === undefined) {
+            const addressField = this.safeValue (response, 'account');
+            address = this.safeValue (addressField, 'address');
+            memo = this.safeValue (addressField, 'memo');
+        }
+        this.checkAddress (address);
+        return {
+            'currency': code,
+            'address': address,
+            'tag': memo,
+            'network': this.safeValue (params, 'network'),
+            'info': response,
+        };
+    }
+
+    async withdraw (code, amount, address, tag = undefined, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code); // check if it has canDeposit
+        const request = {
+            'ticker': currency['id'],
+            'amount': this.numberToString (amount),
+            'address': address,
+        };
+        const uniqueId = this.safeValue (params, 'uniqueId');
```

Comment:
Let's use `this.uuid()` or `this.uuid22()` here to generate it.

Reply:
I've used the uuid generator when the uniqueId isn't being provided